src/store/setToolMode.js

```diff
@@ -8,67 +8,137 @@ import { state } from './../store/index.js';
  * respond to user input, and can create new data
  *
  * @export
- * @param {*} element
- * @param {*} toolName
- * @param {*} options
+ * @param {object} element
```

Comment:
Side note: We need to blitz the whole lib before release and fix all these params, `Object` should be upper case for instance. I've also used `Array` incorrectly throughout, this is how it should be done:
```
{Object}
{string}
{number}
{boolean}
{Object[]}
{string[]}
{number[]}
{boolean[]}
```

Also if something is optional, we put the variable name in brackets after, e.g.:

`@param {(Object|number} [options]`

further reading: [here](http://usejsdoc.org/tags-param.html) and [here](http://usejsdoc.org/tags-type.html).